platform/android/MapboxGLAndroidSDK/src/test/java/org/maplibre/android/location/engine/LocationEngineRequestTest.kt

```diff
@@ -0,0 +1,64 @@
+package org.maplibre.android.location.engine
+
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+class LocationEngineRequestTest {
+    @Test
+    fun checkDefaultValues() {
+        val request = LocationEngineRequest.Builder(1000L).build()
+        Assertions.assertThat(request.interval).isEqualTo(1000L)
```

Comment:
Can't you import `assertThat` like in Java? This is very verbose.

Reply:
see https://github.com/maplibre/maplibre-native/pull/1926#issuecomment-1847334143

Reply:
imported